core/server/worker/src/main/java/alluxio/worker/block/SpaceReserver.java

```diff
@@ -90,6 +90,11 @@ public SpaceReserver(BlockWorker blockWorker) {
             PropertyKey.Template.WORKER_TIERED_STORE_LEVEL_LOW_WATERMARK_RATIO.format(ordinal);
         Preconditions.checkArgument(Configuration.getDouble(tierLowWatermarkProp) >= 0,
             "The low watermark of tier " + ordinal + " should not be negative");
+        Preconditions.checkArgument(
+            Configuration.getDouble(tierLowWatermarkProp) < Configuration
+                .getDouble(tierHighWatermarkProp),
+            "The low watermark of tier " + ordinal
+                + " should not be smaller than the high watermark");
```

Comment:
Define variables for the high and low watermark values? Also, should we output what they are in the warning message?